test-utils/assert/core/src/main/java/org/camunda/bpm/engine/test/assertions/bpmn/BpmnAwareTests.java

```diff
@@ -952,16 +952,42 @@ public static void complete(ExternalTask externalTask, Map<String, Object> varia
     if (externalTask == null || variables == null) {
       throw new IllegalArgumentException(format("Illegal call of completeExternalTask(externalTask = '%s', variables = '%s') - both must not be null!", externalTask, variables));
     }
+    complete(externalTask, variables, Collections.EMPTY_MAP);
+  }
+
+  /**
+   * Helper method to easily fetch, lock and complete an external task.<p>
+   * Note: if multiple external tasks exist that can be locked for the topic of
+   * the given external task, this method might throw an
+   * {@link IllegalStateException} if an external task with a different id is
+   * locked by chance. In this case, it is more advisable to use the
+   * {@link #fetchAndLock(String, String, int) fetchAndLock} and
+   * {@link #complete(LockedExternalTask, Map)} methods to achieve reliable results.
+   *
+   * @param externalTask
+   *          External task to be completed
+   * @param variables
+   *          Process variables to be passed to the process instance when
+   *          completing the task. For setting those variables, you can use
+   *          withVariables(String key, Object value, ...)
+   * @param localVariables
+   *          Local process variables to be passed to the process instance when
+   *          completing the task. For setting those variables, you can use
+   *          withVariables(String key, Object value, ...)
+   */
+  public static void complete(ExternalTask externalTask, Map<String, Object> variables, Map<String, Object> localVariables) {
+    if (externalTask == null || localVariables == null) {
+      throw new IllegalArgumentException(format("Illegal call of completeExternalTask(externalTask = '%s', localvariables = '%s') - both must not be null!", externalTask, localVariables));
+    }
```

Comment:
```suggestion
    if (externalTask == null || (variables == null && localVariables == null)) {
      throw new IllegalArgumentException(format("Illegal call of completeExternalTask(externalTask = '%s', variables = '%s', localVariables = '%s') - provide external task and either variables or local variables.", externalTask, variables, localVariables));
    }
```

Let's fail this explicitly when both `variables` and `localVariables` are `null`. Right now, when `variables` are provided, but `localVariables` are `null`, it would fail.